src/bfs.c

```diff
@@ -0,0 +1,49 @@
+#include "bfs.h"
+
+// Function to initialise the attributes of each node
```

Comment:
[Example from kernel](https://github.com/torvalds/linux/blob/master/drivers/android/binder.c#L81)
[Example from git](https://github.com/git/git/blob/master/attr.h#L17)

I think `/*` is more elegant than `//`.